sketch.js

```diff
@@ -1,15 +1,81 @@
-const container = document.querySelector(".js-grid-container");
-addElement();
-const gridItem = document.querySelector(".js-grid-item");
-container.addEventListener("mouseover", function(e) {
-  const el = e.target;
-  el.classList.add("highlight");
-});
-
-function addElement() {
-  for (i = 0; i < 256; i++) {
-    const newDiv = document.createElement("div");
-    newDiv.classList.add("outline", "grid__item", "js-grid-item");
-    container.appendChild(newDiv);
-  }
-}
+(function() {
+  const container = document.querySelector(".js-grid-container");
+  const resizeButton = document.querySelector(".js-resize-grid");
+  const radioButton = document.querySelector(".js-radio-buttons");
+  let colorMode = 0;
+
+  addElement(16);
+
+  radioButton.addEventListener("change", function(e) {
+    colorMode = e.target.value;
+  });
+
+  container.addEventListener("mouseover", drawPixels);
+
+  resizeButton.addEventListener("click", function(e) {
+    const cellNumber = window.prompt("Enter a number 1-100?");
+    removeGrid();
+    addElement(cellNumber);
+  });
+
+  function addElement(n) {
+    for (i = 0; i < n; i++) {
+      const row = document.createElement("div");
+      row.classList.add("grid__row", "js-grid-row", "js-grid-item");
+      container.appendChild(row);
+      for (j = 0; j < n; j++) {
+        const cell = document.createElement("div");
+        cell.classList.add("grid__cell", "js-grid-item", "js-grid-cell");
+        row.appendChild(cell);
+      }
+    }
+    addDataAttribute();
```

Comment:
Don't need to set the default value. Check for `undefined` when you read the value from the attribute, and use `100` if the stored value is `undefined.